src/NavBar/__snapshots__/NavBar.test.js.snap

```diff
@@ -0,0 +1,78 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
If we change the alt text, then this will need to be updated as well.

Reply:
You're right. Will do.